src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -620,6 +620,29 @@
         </where>
     </sql>
     
+    <!--
+        Convert string values to a single numerical value for filtering purposes only.
+        Not designed as a general purpose decimal number parser.
+        
+        Examples conversions:
+        '6' to 6
+        '>=6' to 6
+        '<=6' to 6
+        '>6' to 6.00..1...
+        '<6' to 5.99..9...
+    -->
+    <sql id="castStringValueToFloat">
```

Comment:
@onursumer @haynescd if it turns out this is a performance killer, we could precompute this right?

Reply:
Yea, you could do this when you create the table